wdl-lint/tests/lints/curly-command/source.errors

```diff
@@ -1,3 +1,11 @@
+note[SectionOrdering]: sections are not in order for task bad
+  ┌─ tests/lints/curly-command/source.wdl:5:6
+  │
+5 │ task bad {
```

Comment:
For a lint like this, do we want the reported span to encompass the whole body of the task? Or just the name? I haven't kept up with how we have handled this historically.

Reply:
I think for this lint we could probably label the task name with "this task contains sections that are out of order" with a secondary label on the span of the keyword for the first out-of-order section we encounter with "this section is out of order".

Reply:
Thanks for the suggestion. I didn't realize that you could label two spans (I suppose I could've looked at the docs). Now it highlights the `task` or `workflow` and then adds a secondary highlight to the first offending section that it encounters.